include/fastdds/rtps/security/accesscontrol/AccessControl.h

```diff
@@ -152,6 +153,16 @@ class AccessControl
             const std::vector<std::string>& partitions,
             EndpointSecurityAttributes& attributes,
             SecurityException& exception) = 0;
+
+    bool set_logger(Logging* logger, SecurityException& /*exception*/) { logger_ = logger; return true; }
+
+protected:
+
+    const Logging* get_logger() { return logger_; }
```

Comment:
This is a pointer to a `const Logging` object. Do you expect all methods on the logger to log stuff will be const? Or should this instead be a `Logging * const`?

Reply:
`Logging::log(...)` is const. At this point other plugins have no other business than using a `const` object.

Reply:
Excellent.